### PR TITLE
ci: fix release pipelines issues

### PR DESCRIPTION
**Pull request by eugene-babichenko**

### CODE REVIEW

**Comment by eugene-babichenko** [comment]
We need to install the `SLACK_WEBHOOK` secret.

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -325,4 +325,31 @@ jobs:
...
+          SLACK_COLOR=success
+          SLACK_MESSAGE=:rocket: Published release ${{ needs.create_release.outputs.release }}.
+          EOF
+      - name: Status is FAILED
+        if: ${{ needs.publish_release.outputs.status != 'success'  }}
+        run: |
+          cat <<EOF >> $GITHUB_ENV
+          SLACK_COLOR=failure
+          SLACK_MESSAGE=:x: Release ${{ needs.create_release.outputs.release }} *FAILED*.
+          EOF
```
Comment (mzabaluev) [inline .github/workflows/release.yml:349]: I'd unify these steps and do all conditional stuff in the shell code.

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -325,4 +325,31 @@ jobs:
...
+      - name: Send notification
+        uses: rtCamp/action-slack-notify@v2
+        env:
+          SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
```
Comment (mzabaluev) [inline .github/workflows/release.yml:353]: Hope this only requires read-only privileges on GitHub.

File: .github/workflows/release.yml (hunk 1 above)
Comment (eugene-babichenko) [reply .github/workflows/release.yml]: Done.

File: .github/workflows/release.yml (hunk 2 above)
Comment (eugene-babichenko) [reply .github/workflows/release.yml:355]: This does not actually access GitHub APIs, it calls hooks.slack.com.

**Comment by eugene-babichenko** [comment]
Configured the secret.

**Pull request merged**